e2e/mocks/consent-message.mocks.ts

```diff
@@ -1,13 +1,15 @@
 export const mockConsentMessage = ({
   walletUserId,
-  partyUserId
+  partyUserId,
+  tokenSymbol
 }: {
   walletUserId: string;
   partyUserId: string;
+  tokenSymbol: 'ICP' | 'TKN';
```

Comment:
Not really sure if it will be expanded or used more than here (and in the other file modified), but does it make sense to create a mock type for `'ICP' | 'TKN'`?

Reply:
It probably won't expand soon because the Juno Docker used for E2E testing supports only ICP; no other ledger is deployed within the container. TKN is the custom test. Therefore, I would suggest we skip this for now?

Reply:
Note: I merge the PR but, we can create the type in another PR if you think it's worth it.